client/express-checkout/blocks/components/express-checkout-container.js

```diff
@@ -27,9 +27,11 @@ const ExpressCheckoutContainer = ( props ) => {
 	};
 
 	return (
-		<Elements stripe={ stripePromise } options={ options }>
-			<ExpressCheckoutComponent { ...props } />
-		</Elements>
+		<div style={ { minHeight: '40px' } }>
```

Comment:
Restoring this because Google Pay wasn't showing with the button set to Large. This line was present before #9258